File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/service/GAV.java
package org.javamaster.invocationlab.admin.service;

import lombok.Data;

/**
 * @author yudong
 */
@Data
public class GAV {
    String groupID;
    String artifactID;
    String version;

    @Override
    public String toString() {
        return this.getGroupID() + ":" + this.getArtifactID() + ":" + this.getVersion();
    }
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/service/GAV.kt
package org.javamaster.invocationlab.admin.service

/**
 * @author yudong
 */

class GAV {
    var groupID: String? = null
    var artifactID: String? = null
    var version: String? = null

    override fun toString(): String {
        return this.groupID + ":" + this.artifactID + ":" + this.version
    }
}
